src/dex/jarvis-v6/config.ts

```diff
@@ -0,0 +1,68 @@
+import { DexParams } from './types';
+import { DexConfigMap, AdapterMappings } from '../../types';
+import { Network, SwapSide } from '../../constants';
+import { Interface } from '@ethersproject/abi';
+import ChainlinkAccessControlledOffchainAggregatorABI from '../../abi/jarvis/ChainlinkAccessControlledOffchainAggregator.json';
+import SynthereumPriceFeedABI from '../../abi/jarvis/SynthereumPriceFeed.json';
+
+export const JarvisV6Config: DexConfigMap<DexParams> = {
+  JarvisV6: {
```

Comment:
only jEUR and jCHF will be swappable from/to ?

Reply:
For now only jEUR and jCHF have been deployed with v6.

Reply:
Missing pools config have been added [here](https://github.com/paraswap/paraswap-dex-lib/pull/171/commits/80b30b2e89e0335a3d1a3533751f787053017bb2)